### ISSUE TITLE
Saved model support for loading and set with different parameters?

### ISSUE THREAD

**Issue by WenchenLi** [dmlc/xgboost#308]
Hi, 
Is it possible to load the saved model with different parameters like learning rate and gamma that doesn't affect the structure of the model? thanks!

**Comment by tqchen** [comment]
Yes, it is possible.
